src/fermilib/utils/_plane_wave_hamiltonian.py

```diff
@@ -164,14 +177,14 @@ def plane_wave_hamiltonian(n_dimensions, grid_length, length_scale, geometry,
 
     if momentum_space:
         return jellium_model(n_dimensions, grid_length, length_scale, spinless,
-                             True) + \
+                             True, energy_cutoff) + \
```

Comment:
Please see Issue #16 opened by @damiansteiger . We are trying get rid of all line conjugation with the "\" symbol. Would you mind removing these from this file prior to merge?